lib/ansible/vars/manager.py

```diff
@@ -151,26 +151,6 @@ def options_vars(self, value):
             raise AnsibleAssertionError("the type of 'value' for options_vars should be a dict, but is a %s" % type(value))
         self._options_vars = value.copy()
 
-    def _preprocess_vars(self, a):
```

Comment:
Is this just simply no longer needed? Doesn't seem related the the "None aware" change above so I wanted to make sure I wasn't missing something.

Reply:
not needed, the code changed above called it at one point but doesn't anymore